### PR TITLE
feat(async/stack): add async command to delete normal stack [EE-4558]

### PR DESCRIPTION
**Pull request by oscarzhou-portainer**
closes [EE-4558]

[EE-4558]: https://portainer.atlassian.net/browse/EE-4558?

### CODE REVIEW

File: edge/poll_async.go [hunk 1]
```diff
@@ -355,3 +360,26 @@ func (service *PollService) processVolumeCommand(command client.AsyncCommand) er
...
+	}
+
+	switch normalStackCommand.StackOperation {
+	case "remove":
+		folder := fmt.Sprintf("%s/%s", command.Path, normalStackCommand.Name)
+		fileName := "docker-compose.yml"
+		fileContent := normalStackCommand.StackFileContent
+		err = filesystem.WriteFile(folder, fileName, []byte(fileContent), 0644)
+		if err != nil {
+			break
```
Comment (chiptus) [inline edge/poll_async.go]: log the error

File: edge/poll_async.go [hunk 2]
```diff
@@ -355,3 +360,26 @@ func (service *PollService) processVolumeCommand(command client.AsyncCommand) er
...
+	if err != nil {
+		return newOperationError("normalStack", "n/a", err)
+	}
+
+	switch normalStackCommand.StackOperation {
+	case "remove":
+		folder := fmt.Sprintf("%s/%s", command.Path, normalStackCommand.Name)
+		fileName := "docker-compose.yml"
+		fileContent := normalStackCommand.StackFileContent
+		err = filesystem.WriteFile(folder, fileName, []byte(fileContent), 0644)
```
Comment (chiptus) [inline edge/poll_async.go]: why do we need this? today to remove a stack we only need its name. if you have the name, the content is ignored anyway

Comment (oscarzhou-portainer) [reply edge/poll_async.go]: Do we not use `docker-compose down -f xx.yml` to remove a stack?

Comment (chiptus) [reply edge/poll_async.go]: no, we use `docker compose -p PROJECT_NAME`

**Review by andres-portainer — APPROVED** [review]

**Pull request merged**